CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and includes an additional section for migration notes.
 
 
 ## [Unreleased]
+- *ORCA-109* request_files now uses SQS queue for recovery status updates, and receives input from a separate SQS queue.
```

Comment:
Should be moved to under the `### CHANGED` section of `##[Unreleased]`

Reply:
Fixed